reactive.js

```diff
@@ -0,0 +1,56 @@
+/* reactive engine. the code here combines functions exposed by the
+ * library into a single function to be run. i keep all code depending
+ * on Bacon here, so if somebody will want to stop using Bacon in the
+ * future, it will be easier */
+var lib = require('./lib2');
+var Bacon = require('bacon');
+
+// this is an adapter function. it gets a Pouch changes object and it
+// returns a Bacon stream
+function changesToStream(changes) {
+  return Bacon.fromBinder(function(sink) {
+    changes
+      .on('change', sink)
+      .on('error', function(text) {
+        sink(new Bacon.Error(text));
+      })
+      .on('complete', function() {
+        sink(new Backn.End());
+      });
+  });
+}
+
+module.expose = function(options) {
+  /* the initial configuration document is fetched from the
+   * database. this is updated when a change is detected */
+  var withOptions = lib.withOptions(options),
```

Comment:
You are beding one var statement quite a lot here. I had to look multiple times to actually get, how the code is structured syntactically. I would prefer multple statements except just having one, because you have to scan up all the way up to check why the statements are comma-seperated. But that is just syntax.